resources/keycloak/custom-user-provider/src/main/java/com/cvmanager/auth/provider/user/pojos/UserObject.java

```diff
@@ -0,0 +1,147 @@
+package com.cvmanager.auth.provider.user.pojos;
+
+import java.sql.ResultSet;
+import java.util.List;
+import java.util.Map;
+import java.util.HashMap;
+
+import org.keycloak.models.UserModel;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+@JsonSerialize
+public class UserObject {
+    private static final Logger log = LoggerFactory.getLogger(OrganizationObject.class);
+
+    @JsonProperty("keycloak_id")
+    private String id;
+    @JsonProperty("user_id")
+    private Integer user_id;
```

Comment:
suggestion(non-blocking): since we're using the `@JsonProperty` annotation to map the json name to the pojo field name, we can use java's naming convention (camel case) instead
```suggestion
    @JsonProperty("user_id")
    private Integer userId;
```

Reply:
updated